lib/fake-xhr/index.js

```diff
@@ -333,7 +333,10 @@ function fakeXMLHttpRequestFor(globalScope) {
 
     function verifyRequestOpened(xhr) {
         if (xhr.readyState !== FakeXMLHttpRequest.OPENED) {
-            throw new Error(`INVALID_STATE_ERR - ${xhr.readyState}`);
+            const errorMessage = xhr.readyState === FakeXMLHttpRequest.UNSENT
+                ? "INVALID_STATE_ERR - you might be trying to set the request state for a request that has been aborted, it is reccomended to check 'readyState' first..."
+                : `INVALID_STATE_ERR - ${xhr.readyState}`;
```

Comment:
I am not too into the implementation details here. Are the previous strings used here from the spec?

Reply:
I found this, but didn't find error related stuff there:
https://xhr.spec.whatwg.org/

Reply:
You are absolutely right, there is no such thing in the spec. Merging this. Thanks!